textBasedAdventure.js

```diff
@@ -1,5 +1,142 @@
+  if (userInputRoomTwo.toLowerCase() === 'play'){
+      userInputRoomTwo = readline.question(`Hello ${playerName}, you took great pleasure in torturing the kids at the fat camp your worked at. It is now time for you to shed some weight of your own. I require a pound of your flesh placed on the scale. The drills you see mounted to the device on your head will continue to inch closer and closer to your head until they scramble your brains. I hope your teeth are a sharp as your words, because there is no knife. Do you begin to chew away at your flesh or make your way to the door. Please type "chew" or "door"`)
+        
+        if (userInputRoomTwo.toLowerCase() === 'chew'){
+            userInputRoomTwo = readline.question('You slip off the handcuffs and stare at your bare arm. You take a deep breath and take a huge bite of your arm. Pain shoots through your arm as you tear the flesh away and the taste of your own blood fills your mouth. You begin to gag. Do you keep continue or do you head for the door? Please type "continue" or "door"')
+
+              if (userInputRoomTwo.toLowerCase() === 'continue'){
+                  console.log(`You continue to tear away pieces of your arm and pile them up on the scale as you can hear the drill whirring. Just as the drill makes contact with the sides of your head you place the final piece of your arm on the scale and the whirring comes to a halt. The device falls to the ground and you hear the door unlock. You walk through the door and open another door that leads to a back alley way. You are in the middle of the city at what appears to be midnight. You have survived the ordeal. Congratulations ${playerName}, live your new life with humility or you will find yourself in an unwinnable game.`)
+                  winCount++
+              } else if (userInputRoomTwo.toLowerCase() === 'door'){
+                  console.log('You run for the door and as you approach you begin to hear a rapid beeping. The drills begin to spin at full speed and drive straight into your skull. You collapse in front of the door and your body is dragged away into the darkness by a man in a pig mask. The game master always gets their way. GAME OVER!')
+                  deathCount++
+              } else {
+                  console.log(`Defiance is not tolerated ${playerName}. A man in a pig mask emerges and straps you back to the chair. He proceeds to slowly flay the flesh from your body as the drill continues to inch closer and closer to your skull. When the drills finally put you out of your misery he has cut both of your arms and one of your legs to the bone. GAME OVER! `)
+                  deathCount++
+              }
+        } else if (userInputRoomTwo.toLowerCase === 'door'){
```

Comment:
You forgot to invoke the method.

-5 HP

Tip: If you want to control the user's input,  you can try to convert their input to lowerCase as soon as they make their choice so you dont have to check in every case.